### PR TITLE
Issue 1025 - Convert Check for Understanding Into Questions ( Part V )

### PR DESCRIPTION
**Pull request by HenriqueFadoni**
Convert Check for Understanding Into Questions ( Part V ) #1025 

- Split questions in bullet points
- Added 3 questions
- Properly edited one question that was bad formatted

### CODE REVIEW

File: mobile-development/mobile-development.md [hunk 1]
```diff
@@ -91,4 +91,7 @@ Work through this React Native tutorial:
 
 ### Check for Understanding
 
-Find a partner who you haven't worked with today, and spend 5-10 minutes each explaining how you converted your project.
+Find a partner who you haven't worked with today, and discuss about:
+
+1- In 5-10 minutes each explain how you converted your project.
```
Comment (alodahl) [inline mobile-development/mobile-development.md]: Please cover all of the points In the Objectives section above.

File: roles-in-tech/developer-advocate.md [hunk 2]
```diff
@@ -182,6 +182,13 @@ Perform the Group Disscussion among yourself. Group Disscussion can be on any to
 Practice soft skills by regular interaction with different people.
 Attend Conferences and meetups of different community and share the experience with each other.
 
+### Check for Understanding
+
+In groups of at least 2 people, answer:
+
+1- What are the main advantages and skills of a Developer Advocate?
```
Comment (alodahl) [inline roles-in-tech/developer-advocate.md]:
```suggestion
1. What are the main advantages and skills of a Developer Advocate?
```

File: roles-in-tech/developer-advocate.md [hunk 3]
```diff
@@ -182,6 +182,13 @@ Perform the Group Disscussion among yourself. Group Disscussion can be on any to
...
+2- What are some of the Developer Advocate's responsabilities?
```
Comment (alodahl) [inline roles-in-tech/developer-advocate.md]:
```suggestion
1. What are some of the Developer Advocate's responsibilities?
```

**Comment by HenriqueFadoni** [comment]
I would love to help you more. However, it seems this repository is not eligible for hacktoberfest :/

**Comment by alodahl** [comment]
What makes you say that?

**Comment by HenriqueFadoni** [comment]
When I first Log in at Hacktobersfest's account it says that my PR is no valid since:

" 
Ineligible Repository

Your PR was submitted to a repository that we've excluded from Hacktoberfest as it doesn't align with our core values.
"

Which also makes me confused haha

**Comment by HenriqueFadoni** [comment]
My PR was also labeled as '' Invalid '' by an admin. This also makes me a lot confused inasmuch as it is my first PR and I am correcting an issue.

**Comment by alodahl** [comment]
I haven’t had to mark any PRs as invalid. I dont think that one was with Techtonica.

**Comment by alodahl** [comment]
Oh, your PR has to be merged before you get points for it.

**Comment by alodahl** [comment]
Henrique, I just merged my own PR to the project and got a Hacktoberfest point... your issues must be happening in another repo, or the "invalid" mark just means I requested changes.

**Comment by HenriqueFadoni** [comment]
Oh ok, no worries. I will be finishing the PR today

**Comment by alodahl** [comment]
Thanks for your help!

**Comment by HenriqueFadoni** [comment]
> Thanks for your help!

You are welcome! ( I still have some trouble with Hacktoberfest )

**Pull request merged**